store/tikv/lock_resolver.go

```diff
@@ -566,7 +583,13 @@ func (lr *LockResolver) getTxnStatus(bo *Backoffer, txnID uint64, primary []byte
 			return status, err
 		}
 		status.action = cmdResp.Action
-		if cmdResp.LockTtl != 0 {
+		if cmdResp.LockInfo != nil {
+			status.secondaries = cmdResp.LockInfo.Secondaries
+			status.useAsyncCommit = cmdResp.LockInfo.UseAsyncCommit
+		}
+		if status.useAsyncCommit {
+			status.commitTS = cmdResp.LockInfo.MinCommitTs
+		} else if cmdResp.LockTtl != 0 {
```

Comment:
This means we deliberately set `LockTtl` to 0 for async commit txn. We might resolve async commit locks too eagerly without any backoff.